src/main/java/com/google/devtools/build/lib/query2/cquery/ConfiguredTargetAccessor.java

```diff
@@ -131,8 +131,8 @@ public boolean isTestSuite(ConfiguredTarget target) {
       throw new QueryException(
           caller,
           String.format(
-              "%s %s of type %s does not have attribute '%s'",
-              errorMsgPrefix, actualConfiguredTarget, rule.getRuleClass(), attrName),
+              "%sconfigured target of type %s does not have attribute '%s'",
```

Comment:
does this need a space between %s and configured ? It looks fine in the test but the error message above does seem to have a space after errorMsgPrefix.

Reply:
No, the passed-in prefix already has the space, and I didn't want to try and fix all the other uses to make it look nicer here.